patches/chrome-VERSION.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/chrome/VERSION b/chrome/VERSION
-index 3d9a560e26e43fb2da1332dff339c1039cfb1a3e..7b2d1041533f1aacdeefb8524a4c433f390091b0 100644
```

Comment:
We _don't_ need this patch in this PR though, right...?

Reply:
I can remove it 😄 It just came up when I did `npm run update-patches`